lttngpy/src/lttngpy/channel.cpp

```diff
@@ -12,12 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Include lttng-error.h before channel.h, otherwise lttng_error_code isn't known
+// TODO: order it alphabetically once liblttng-ctl 2.14+ is fixed:
```

Comment:
cpplint is complaining about a missing username for the TODO: https://build.ros2.org/job/Rpr__ros2_tracing__ubuntu_noble_amd64/205/testReport/junit/lttngpy/cpplint/readability_todo__2____tmp_ws_src_ros2_tracing_lttngpy_src_lttngpy_channel_cpp_16_/

> Missing username in TODO; it should look like "// TODO(my_username): Stuff."

You can add your username or mine if you prefer.